locations/spiders/gsf_car_parts_gb.py

```
@@ -2,47 +2,33 @@
 from locations.structured_data_spider import StructuredDataSpider
+from locations.user_agents import BROWSER_DEFAULT
 
 
-class GsfCarPartsGBSpider(SitemapSpider, StructuredDataSpider):
+class GsfCarPartsGBSpider(CrawlSpider, StructuredDataSpider):
     name = "gsf_car_parts_gb"
     item_attributes = {"brand": "GSF Car Parts", "brand_wikidata": "Q80963064"}
-    sitemap_urls = ["https://www.gsfcarparts.com/robots.txt"]
-    sitemap_follow = ["branch-sitemap"]
-    sitemap_rules = [(r"/branches/([-\w]+)$", "parse")]
+    start_urls = ["https://www.gsfcarparts.com/branches"]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)